source/isaaclab/isaaclab/envs/manager_based_env.py

```diff
@@ -190,6 +190,13 @@ def __init__(self, cfg: ManagerBasedEnvCfg):
         if self.cfg.export_io_descriptors:
             self.export_IO_descriptors()
 
+        # show deprecation message for rerender_on_reset
+        if self.cfg.rerender_on_reset:
```

Comment:
You can throw the warning and then internally set the variable `num_rerenders_on_reset` to 1. This then simplifies your check at other places. Also could you use deprecate warning from python instead of omni logger: https://docs.python.org/3/library/warnings.html